src/cli/README_NETDATA.md

```diff
@@ -244,7 +244,7 @@ Done
 
 ### show
 
-Usage: `netdata show [-x]`
+Usage: `netdata show [local] [-x]`
```

Comment:
Agree it's a good idea to add `local` flag for `netdata show` command.

We may also follow the same pattern for `route` and `prefix` commands?

Reply:
Both `prefix` and `route` commands only view and manipulate the local network data. Can you provide more detail on what you are suggesting?

Reply:
The problem with `prefix` and `route` commands is that users, without reading CLI command doc carefully, usually think that the commands should print the effective prefixes and routes in the Network Data.
I think it's quite understandable since the local Network Data is actually an implementation detail rather than a Thread protocol concept. 
So, a less confusing way of implementing `prefix` and `route` maybe:
- `prefix` and `route` prints the prefixes/routes in Network Data
- `prefix local` and `route local` prints the prefixes/routes in local Network Data

Reply:
But to be consistent, would you also change the existing `prefix add` to something like `prefix local add`? Then have `prefix add` return error? One downside is that it will break existing scripts in doing so.

Reply:
> But to be consistent, would you also change the existing prefix add to something like prefix local add?

No, it should just be `prefix add`. 

Fine to keep `prefix` and `route` unchanged.